python_apps/pypo/pypo/recorder.py

```diff
@@ -152,10 +153,10 @@ def set_metadata_and_save(self, filepath):
             recorded_file['title'] = "%s-%s-%s" % (self.show_name,
                     full_date, full_time)
             #You cannot pass ints into the metadata of a file. Even tracknumber needs to be a string
-            recorded_file['tracknumber'] = unicode(self.show_instance)
+            recorded_file['tracknumber'] = str(self.show_instance)
```

Comment:
I assume the string conversion is not needed as `self.show_instance` is a string already.